cluster/allinone.sh

```diff
@@ -18,7 +18,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-FRAKTI_VERSION="v1.9"
+FRAKTI_VERSION="v1.9.1"
```

Comment:
The release version in deployment doc should also update to `1.9.1` instead of `1.9`